ANTLR/BaseVisitor.py

```diff
@@ -140,6 +154,16 @@ def visitRemove_all(self, ctx: GoPoParser.Remove_allContext):
         self.tmp_memory['list'] = list(filter(lambda e: e != number_to_remove, self.tmp_memory['list']))
         return self.visitChildren(ctx)
 
+    def visitReverse(self, ctx:GoPoParser.ReverseContext):
+        self.tmp_memory['list'].reverse()
+        return self.visitChildren(ctx)
+
+    def visitSort(self, ctx:GoPoParser.SortContext):
+        is_descending = False if self.visit(ctx.getChild(2)) == '+' else True
```

Comment:
Similar to filter. Maybe it will be better to add an **op** field by a little change in g4 file?

Changed .g4 file
`sort		: SORT OPAR op=(PLUS | MINUS) CPAR;`

After change you can easily do something like:

```
is_descending = False if ctx.op.type == GoPoParser.PLUS else True
```

Reply:
done